src/reactjs_jsx_ppx_v4.ml

```diff
@@ -0,0 +1,1455 @@
+("alt", {jsString = None; typeString = mkPropConst (Lident "string")});
+("async", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("autoComplete", {jsString = None; typeString = mkPropConst (Lident "string")});
+("autoCapitalize", {jsString = None; typeString = mkPropConst (Lident "string")});
+("autoFocus", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("autoPlay", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("challenge", {jsString = None; typeString = mkPropConst (Lident "string")});
+("charSet", {jsString = None; typeString = mkPropConst (Lident "string")});
+("checked", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("cite", {jsString = None; typeString = mkPropConst (Lident "string")});
+("crossOrigin", {jsString = None; typeString = mkPropConst (Lident "string")});
+("cols", {jsString = None; typeString = mkPropConst (Lident "int")});
+("colSpan", {jsString = None; typeString = mkPropConst (Lident "int")});
+("content", {jsString = None; typeString = mkPropConst (Lident "string")});
+("controls", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("coords", {jsString = None; typeString = mkPropConst (Lident "string")});
+("data", {jsString = None; typeString = mkPropConst (Lident "string")});
+("dateTime", {jsString = None; typeString = mkPropConst (Lident "string")});
+("default", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("defer", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("disabled", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("download", {jsString = None; typeString = mkPropConst (Lident "string")});
+("encType", {jsString = None; typeString = mkPropConst (Lident "string")});
+("form", {jsString = None; typeString = mkPropConst (Lident "string")});
+("formAction", {jsString = None; typeString = mkPropConst (Lident "string")});
+("formTarget", {jsString = None; typeString = mkPropConst (Lident "string")});
+("formMethod", {jsString = None; typeString = mkPropConst (Lident "string")});
+("headers", {jsString = None; typeString = mkPropConst (Lident "string")});
+("height", {jsString = None; typeString = mkPropConst (Lident "string")});
+("high", {jsString = None; typeString = mkPropConst (Lident "int")});
+("href", {jsString = None; typeString = mkPropConst (Lident "string")});
+("hrefLang", {jsString = None; typeString = mkPropConst (Lident "string")});
+("htmlFor", {jsString = None; typeString = mkPropConst (Lident "string")});
+("httpEquiv", {jsString = None; typeString = mkPropConst (Lident "string")});
+("icon", {jsString = None; typeString = mkPropConst (Lident "string")});
+("inputMode", {jsString = None; typeString = mkPropConst (Lident "string")});
+("integrity", {jsString = None; typeString = mkPropConst (Lident "string")});
+("keyType", {jsString = None; typeString = mkPropConst (Lident "string")});
+("kind", {jsString = None; typeString = mkPropConst (Lident "string")});
```

Comment:
According to [WHATWG HTML living standard](https://html.spec.whatwg.org/#attr-track-kind), this should be an enumerated attribute.
```suggestion
("kind", {jsString = None; typeString = mkPropVariant ["subtitles"; "captions"; "descriptions"; "chapters"; "metadata"]});
```